java-squid/src/main/java/org/sonar/plugins/java/api/tree/NewClassTree.java

```diff
@@ -75,4 +76,6 @@
   @Nullable
   ClassTree classBody();
 
+  Symbol constructor();
```

Comment:
I would go for constructorSymbol here. In other places of the API symbol is refered with `symbol` name.